app/enterprise/0.34-x/developer-portal/how-to/serve-assets/aws-cloudfront.md

```diff
@@ -0,0 +1,56 @@
+---
+title: How to Use AWS Cloudfront to Serve Kong Dev Portal Assets
+toc: false
+---
+
+![AWS Cloudfront](https://s3.us-east-2.amazonaws.com/kong-doc-assets/developer-portal/how-to/aws-cloudfront/aws-cloudfront.jpg)
+
+### Introduction
+
+Amazon CloudFront is a fast content delivery network (CDN) service that securely delivers data, videos, applications, and APIs to customers globally with low latency, high transfer speeds.
+
+In this guide we will create an Amazon S3 bucket, and an Amazon CloudFront distribution to serve static assets for your Kong Developer Portal.
+
+
+### Prerequisites
+
+AWS Account Credentials (with permissions to create S3 buckets, and CloudFront distributions)
+
+
+## Step 1 - Create a new Amazon S3 Bucket
```

Comment:
We should probably reduce these to be either `h3`, or, use the numbered step system from other pages

Reply:
This was done for SEO purposes as "Introduction" and "Prereqs" aren't as important to the purpose of the page as the Steps

Reply:
Digital Ocean places the "Prerequisites" at the same level as the steps. I'd argue that the Introduction serves as the description so it should be the same level as well, that is merely from viewing it as a SEO description.

Regardless, the main point is that this page is hard to read due to the choice of placing the steps in the headings and lack of links to relevant content that were included in the original document.